mtgjson5/providers/tcgplayer.py

```diff
@@ -257,9 +257,7 @@ def generate_mtgjson_sealed_product_objects(
                 sealed_product.release_date = sealed_product.release_date[0:10]
             sealed_product.raw_purchase_urls[
                 "tcgplayer"
-            ] = "https://shop.tcgplayer.com/product/productsearch?id={}&utm_campaign=affiliate&utm_medium=api&utm_source=mtgjson".format(
-                sealed_product.identifiers.tcgplayer_product_id
-            )
+            ] = f"https://shop.tcgplayer.com/product/productsearch?id={sealed_product.identifiers.tcgplayer_product_id}&utm_campaign=affiliate&utm_medium=api&utm_source=mtgjson"
```

Comment:
this seems an unrelated cosmetic change

Reply:
Yes, yes it is. Small change, no relevance to anything.

Reply:
it just makes review harder 😅